### PR TITLE
random fixes and tweaks

### PR DESCRIPTION
**Pull request by GDLW24**
- added pins for strong fatorays
- fixed some tiles being spaced on xenoarch
- remade the gluttony ruin
- removed the WG culture bottle
- removed some belly sprite placeholders
- added fatorays and fatnades to the uplink
- updated fat thresholds in pref menu
- changed calorite statue descriptions slightly
- renamed asimov ai laws to gato crew
- added flowers to loadout menu
- changed a small description in ghosting cryo menu
- fixed some medbay metastation stuff
- fixed snack store not having power
- fixed a typo in raygun scraps
- added WG-related ion laws
- more screen alerts for fatness
- more icons (thanks Static for helping)

### CODE REVIEW

**Review by surftheseawing — COMMENTED** [review]
Just some minor spelling and grammar.

File: code/_onclick/hud/alert.dm [hunk 1]
```diff
@@ -172,11 +183,23 @@
...
+
+/obj/screen/alert/fatter
 	name = "Fat"
-	desc = "You ate too much food, and now your body has plumped up, let's hope your clothes don't rip."
+	desc = "Clothes begin to cling onto your frame tighter and tighter, as your body begins to display first signs of fatness. It's still no big deal, right?"
+	icon_state = "fatter"
+
+/obj/screen/alert/fat
+	name = "Rounded"
+	desc = "Your body's beginning to display the first signes of softened curves, though they're barely noticeable."
```
[the remark this answers is not in the document]
Comment (GDLW24) [reply code/_onclick/hud/alert.dm:198]: Argh, I should look over my tweaks more carfully. Thanks!

File: code/_onclick/hud/alert.dm [hunk 2]
```diff
@@ -172,11 +183,23 @@
 	desc = "Your body is swollen with lard, and it's challenging to move. Despite this, there's something... pleasing about being so large."
 	icon_state = "obese"
 
-/obj/screen/alert/fat
+/obj/screen/alert/veryfat
+	name = "Very fat"
+	desc = "You're beginning to inch closer and closer to proper obesity. The way your thighs rub against each and the rumble your stomach produces suggests that you won't remain in this state for long..."
```
[the remark this answers is not in the document]
Comment (GDLW24) [reply code/_onclick/hud/alert.dm:188]: Ope, it was supposed to be "the rumble of your stomach". Thanks

[the remark this answers is not in the document]
Comment (GDLW24) [reply code/_onclick/hud/alert.dm:188]: Thanks!

**Pull request merged**